ghc-lib-gen.cabal

```diff
@@ -1,47 +1,46 @@
-cabal-version:       2.0
-name:                ghc-lib-gen
-version:             0.1.0.0
-synopsis:            Cabal file generator for GHC as a library
-description:         Generate a cabal file for a subset of GHC for use as a library.
-homepage:            https://github.com/digital-asset/ghc-lib
-license:             BSD3
-x-license:           BSD-3-Clause OR Apache-2.0
-license-file:        LICENSE
-author:              Shayne Fletcher (shayne.fletcher@digitalasset.com)
-maintainer:          Shayne Fletcher (shayne.fletcher@digitalasset.com)
-copyright:           Digital Asset 2018-2019
-category:            Development
-build-type:          Simple
+cabal-version: 3.4
+name: ghc-lib-gen
+version: 0.1.0.0
+synopsis: Cabal file generator for GHC as a library
+description: Generate a cabal file for a subset of GHC for use as a library.
+homepage:https://github.com/digital-asset/ghc-lib
+license: BSD-3-Clause
+x-license: BSD-3-Clause OR Apache-2.0
+license-file: LICENSE
+author: Shayne Fletcher (shayne.fletcher@digitalasset.com)
+maintainer: Shayne Fletcher (shayne.fletcher@digitalasset.com)
```

Comment:
Should the email address here be updated?

Reply:
fixed. thanks @garyverhaegen-da